src/game/client/neo/ui/neo_hud_message.h

```diff
@@ -19,6 +19,8 @@ class CNEOHud_Message : public CNEOHud_ChildElement, public CHudElement, public
 
     void ShowMessage(wchar_t* message);
     void HideMessage();
+    void ShowSubMessage(wchar_t* message);
+    void HideSubMessage();
```

Comment:
`ShowMessage` and `ShowSubMessage` should preferably have their argument as type `const wchar_t*`